tools/proteore_reactome/reactome_analysis.xml

```diff
@@ -0,0 +1,128 @@
+                <param name="file" type="data" format="txt, tabular" label="Input file containing your IDs" help="Input file is a tab-delimited file with at least one contain containing identifier" />
+                <param name="header" type="boolean" checked="true" truevalue="true" falsevalue="false" label="Does file contain header?" />
+		        <param type="text" name="ncol" value="c1" label="Column number of IDs (e.g : Enter c1 for column n°1)">
+                    <validator type="regex" message="Please enter a column number, for example: 'c1' for the first column">[c]{0,1}[0-9]+</validator>
+                </param>
+            </when>
+        </conditional>
+        <param name="species" type="select" label="Species (human, mouse, rat)">
+            <option value="48887">Human (Homo sapiens)</option>
+            <option value="48892">Mouse (Mouse musculus)</option>
+            <option value="48895">Rat (Rattus norvegicus)</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data name="output" format="html" label="" />
+        <data name="trash" format="tabular" hidden="true" label="Invalid id not send to reactome">
+            <discover_datasets pattern="(Reactome_Analysis_Tools_invalid_ID).txt" ext="tabular" visible="false" assign_primary_output="false"/>
+        </data>
+    </outputs>
+    <tests>
+        <test>
+            <conditional name="opt" >
+                <param name="input" value="file" />
+                <param name="file" value="ID_Converter_FKW_Lacombe_et_al_2017_OK.txt" />
+                <param name="header" value="true" />
+                <param name="ncol" value="c1" />
+            </conditional>
+            <output name="output" file="Reactome_Analysis_Tools.html" ftype="html" />
+            <output name="trash" file="Reactome_Analysis_Tools_invalid_ID.txt" ftype="tabular"  />
+        </test>
+    </tests>
+    <help><![CDATA[
+
+**Description**
+
+This tool allows to map your gene/protein list to the Reactome pathway browser (visualize and interact with biological pathways)
+via a web service with pre-loaded data from your list of IDs.
+
+-----
+
+**Input** 
+
+A list of gene/protein IDs. List of IDs can be entered either in a copy/paste manner or by selecting a file with your IDs. 
+
+ID types allowed are: UniProt accNumber (e.g. O75391), Entrez gene ID (e.g. 8086) and gene name (e.g. TP53, NUP58...). If you have other type of ID, please use the ID_Converter tool of ProteoRE.
+
+.. class:: warningmark
+	
+In copy/paste mode, the number of IDs considered in input is limited to 5000.
+
+-----
+
+**Parameters**
+
+"Select species": can be Human (Homo sapiens), Mouse (Mouse musculus) or Rat (Rattus norvegicus)
+
+-----
+
+**Output**
+
+To access results, click on the "View data" icon (from the item created in your history panel), then click on the "Analyze" button in the central panel will directly display the results in the Reactome pathway browser.  
+
+-----
+
+.. class:: infomark
```

Comment:
I would move all those referecenes, including yours into the citation section, this has many advantages in Galaxy including a automatic bibtex report for the user.